juriscraper/pacer/email.py

```diff
@@ -534,116 +534,55 @@ def _get_docket_entries(
-            short_description = re.sub(regex, "", short_description)
-            separator = ";" if self.court_id == "arwb" else "-"
-            short_description = short_description.rsplit(separator, 1)[0]
-        elif self.court_id == "nysb":
-            # In: 22-22507-cgm Ch13 Affidavit Re: Gerasimos Stefanitsis
-            # Out: Affidavit
-            short_description = subject.split(case_name)[0]
-            short_description = short_description.replace("Re:", "")
-            short_description = short_description.split(docket_number)[-1]
-
-            # Remove strings starting with "Ch" followed by a number
-            regex = r"\bCh\d+\b"
-            short_description = re.sub(regex, "", short_description)
-
-            # Remove docket number traces "-AAA"
-            regex = r"^-.*?\s"
-            short_description = re.sub(regex, "", short_description)
-        elif self.court_id in ["pawb", "ndb", "deb", "pamb", "nhb"]:
-            # In: Ch-7 22-20823-GLT U LOCK INC Reply
-            # Out: Reply
-            if case_name in subject:
-                short_description = subject.split(case_name)[-1]
-            elif case_name[:18] in subject:
-                # See deb_2.txt, pamb_1 and pamb_3 for examples
-                short_description = subject.split(case_name[:18])[-1]
-            elif (
-                " and " in case_name and case_name.split(" and ")[0] in subject
-            ):
-                # See pamb_2.txt
-                short_description = subject.split(case_name.split(" and ")[0])[
-                    -1
-                ]
-        elif self.court_id in [
-            "tnmb",
-        ]:
-            # In: Docket Order - Continue Hearing (Auto) Ch 13 Jeffery Wayne Lovell and Tiffany Nicole Lovell 1:24-bk-01377
-            # Out: Docket Order - Continue Hearing (Auto) Ch 13
-            if case_name in subject:
-                short_description = subject.split(case_name)[0]
+        for part in self.docket_numbers + self.case_names:
+            subject = subject.replace(part, " ")
+
+        # Sometimes the full case name is not used in the `subject`
+        # Some courts use a 18 character limit
+        # See deb_2.txt, pamb_1 and pamb_3 for examples
+        for case_name in self.case_names:
+            subject = subject.replace(case_name[:18].strip(), " ")
+            subject = subject.replace(case_name.split(" and ")[0], " ")
+
+        # Deletes:
+        # - extra docket number 'components', such as `federal_dn_judge_initials_assigned`
+        #     these are usually 3 letters long. However, we want to keep some special acronyms
+        #     such as MOR (Merchant of Record?)
+        # - "NEF: " placeholder
+        component_regex = r"((?!-MOR)(\-[A-Z]{2,}))|(\-[a-z]{2,})|(NEF:? )"
+        if self.court_id in ["paeb", "pamb"]:
+            # keeps the "Chapter ..." description on the short description
+            chapter_regex = r"(C[Hh][- ]?(13|7|9|11))|(C[hH][\s-]*$)"
```

Comment:
Without looking too much I was able to find a couple of examples where the removal of `Chapter...` is not compatible.

For instance this one:
[cacb_2.txt](https://github.com/user-attachments/files/18234616/cacb_2.txt)
![Screenshot 2024-12-23 at 7 10 00 p m](https://github.com/user-attachments/assets/cd2645d9-1bf0-4872-aa15-95dc351d583e)

You can see that in the Docket history report, many entries retain "Chapter" in their short descriptions.

Another example comes from `pawb`:

![Screenshot 2024-12-23 at 7 11 59 p m](https://github.com/user-attachments/assets/9a6b03b3-2f45-4685-9828-6b33e1605352)

It seems difficult to determine when `Chapter...` should be removed. I’m also not sure if it's a consistent behavior across courts. If it should sometimes be removed and sometimes not in the same court that's a problem unless we can found a different pattern to decide. However, if it is consistent, we should look for more courts where it should be retained and add them to the condition.